tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
-    "types": ["react", "jest", "webpack-env"],
+    "types": ["react", "jest", "webpack-env", "@testing-library/jest-dom"],
```

Comment:
I got an error message about toBeInDocument(). (Property 'toBeInTheDocument' does not exist on type 'Matchers ') According to the [internet](https://stackoverflow.com/questions/57861187/property-tobeinthedocument-does-not-exist-on-type-matchersany) I needed to run npm i @testing-library/jest-dom --save-dev and then add "@testing-library/jest-dom" to the tsconfig.json.

Reply:
@SireeKoolenWijkstra When do you get this error? I cannot reproduce it. 
I am bit sceptical with changes like this because you're not 100% sure what it does. "according to the internet" and share a link to StackOverflow of more then 3 years old does not really count for a solid argumentation in my opinion.